.github/workflows/library-ci.yml

```diff
@@ -21,27 +21,6 @@ jobs:
         working-directory: packages/browser
       - run: git diff --exit-code # fail if e.g. the mangled properties list has changed, see rollup.config.js
 
-  integration:
```

Comment:
two significant changes here

1. no longer running these tests on push to main

 i think we should still run them, just to catch the very rare risk of things being valid in the PR but not once merged

2. we've gone from two e2e suites - one of which uses posthog cloud - to one e2e suite without that distinction

 it'd be good and my instinct is faster to run them as two suites. the replay ones don't hit posthog cloud iirc and would be good to keep them clearly separate